tidb-cloud/monitor-datadog-integration.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Integrate TiDB Cloud with Datadog (Preview)
+title: Integrate TiDB Cloud with Datadog 
 summary: Learn how to monitor your TiDB cluster with the Datadog integration.
 ---
 
-# Integrate TiDB Cloud with Datadog (Preview)
+# Integrate TiDB Cloud with Datadog 
 
-TiDB Cloud supports Datadog integration (Preview). You can configure TiDB Cloud to send metrics about your TiDB clusters to [Datadog](https://www.datadoghq.com/). After that, you can view these metrics in your Datadog dashboards directly.
+TiDB Cloud supports Datadog integration. You can configure TiDB Cloud to send metrics about your TiDB clusters to [Datadog](https://www.datadoghq.com/). After that, you can view these metrics in your Datadog dashboards directly.
 
 ## Datadog integration version
 
-TiDB Cloud has supported Datadog integration (Beta) since March 04, 2022. Starting from July 31, 2025, TiDB Cloud introduces an enhanced preview version of the integration.
+TiDB Cloud has supported project-level Datadog integration (Beta) since March 4, 2022. Starting from July 31, 2025, TiDB Cloud introduces a preview version of cluster-level Datadog integration. Starting from September 30, 2025, the cluster-level Datadog integration becomes generally available (GA).
```

Comment:
```suggestion
TiDB Cloud has supported project-level Datadog integration (Beta) since March 4, 2022. Starting from July 31, 2025, TiDB Cloud introduces the cluster-level Datadog integration (Preview). Starting from September 30, 2025, the cluster-level Datadog integration becomes generally available (GA).
```